cmd/helm/helm_test.go

```diff
@@ -57,7 +58,8 @@ func TestPluginExitCode(t *testing.T) {
 		cmd.Stdout = stdout
 		cmd.Stderr = stderr
 		err := cmd.Run()
-		exiterr, ok := err.(*exec.ExitError)
+		var exiterr *exec.ExitError
+		ok := errors.As(err, &exiterr)
 
 		if !ok {
```

Comment:
```suggestion
		if !errors.As(err, &exiterr) {
```

This one could be cleaner/tidied I think